### ISSUE TITLE
Revision tab: Focus order incorrect - visual order does not match keyboard tabbing order

### ISSUE THREAD

**Issue by qanitah** [mmiron-HC/WST-RSAMS#90]
URL: https://hcdev.openplus.ca/en/node/1109/revisions/view/11648/14818/visual_inline
User Agent: Chrome, Firefox
Language: Both

Issue Description: When comparing 2 revisions, the top section has a content which is not in correct tabbing order for keyboard user (tabbing order does not match visual order). It goes to Previous change -> Next change. Please see screenshot of the order.

This is the current order:
![image](https://user-images.githubusercontent.com/14280062/129414864-0737b38e-7b50-45fc-b605-a8227f9c0e1c.png)

Steps to reproduce: Select 2 dates to compare so that it has Next change and Previous change in Navigation column. Use keyboard to tab through it.

Expected results: Keyboard tabbing order should be the same as visual order. Next change should receive focus and then the content of it's row. Previous change should come next and then the content of its row. This is the expected order. 

![image](https://user-images.githubusercontent.com/14280062/129414921-692068fe-2331-4ac2-bf14-62d1bb6e2571.png)

**Comment by davidgotlieb** [comment]
OpenPlus has reported the issue to the owner of the module
